language/testing-infra/e2e-tests/src/common_transactions.rs

```diff
@@ -72,6 +72,95 @@ pub static EMPTY_SCRIPT: Lazy<Vec<u8>> = Lazy::new(|| {
         .expect("Failed to compile")
 });
 
+pub static MULTI_AGENT_SWAP_SCRIPT: Lazy<Vec<u8>> = Lazy::new(|| {
+    let code = "
+    import 0x1.DiemAccount;
```

Comment:
```suggestion
    import 0x1::DiemAccount;
```

Does the . syntax still work?

Reply:
I guess the e2e tests are still written in Move IR :(

Reply:
Yes. I think in mvir we have to use `.`?

Reply:
@runtian-zhou/@vgao is there a workaround for this? I feel like I've seen tests like this that use Move source code before (maybe in the testnet cli?).